### DISCUSSION TITLE
How to access the Crossplane UXP UI for monitoring the crossplane resources

### DISCUSSION THREAD

**Discussion by shwethaganiga** [crossplane/crossplane#5545] [Q&A] [answered]
I have installed CrossPlane UXP in the EKS Cluster and I am not finding way to access to the UI of it. Please help me here. Below is the doc I refered:
https://github.com/upbound/universal-crossplane

**Comment by shwethaganiga** [comment]
@all
Could you please help me on the above

**Reply by jeanduplessis** [reply]
@shwethaganiga which UI are you referring to? Are you referring to the Upbound Console? https://www.upbound.io/product/upbound

**Comment by shwethaganiga** [comment]
Hi @jeanduplessis 

Yes I am referring to the Upbound console.

**Reply by jbw976** [reply, marked as answer]
@shwethaganiga thanks for your interest on this topic. It is best fit for the UXP repo (https://github.com/upbound/universal-crossplane), so after this response I will close the issue here and any further discussion can take place there please 🙇‍♂️ 

The Upbound Console does not support standalone UXP instances. The way to achieve what I believe you're looking for is with Upbound Spaces: https://docs.upbound.io/spaces/

When you have a Spaces instance set up, you can then attach it to the Upbound Console using these instructions: https://docs.upbound.io/spaces/attach-detach/

You are welcome to follow up in https://github.com/upbound/universal-crossplane, or feel free to ping me @jbw976 on the [Crossplane Slack](https://slack.crossplane.io/) if you want to chat further with an Upbound team member 😉
